server/src/metadata/field-metadata/utils/field-metadata.util.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { FieldMetadataType } from 'src/metadata/field-metadata/field-metadata.entity';
+
+import {
+  generateTargetColumnMap,
+  convertFieldMetadataToColumnActions,
+} from './field-metadata.util';
+
+describe('generateTargetColumnMap', () => {
+  it('should generate a target column map for a given type', () => {
+    const textMap = generateTargetColumnMap(
+      FieldMetadataType.TEXT,
+      true,
```

Comment:
I would expect _name with customObject set to true

Reply:
If it's a custom object we don't need to specify that's a custom field you are trying to add. You only want the _ if it's for a standard object

Reply:
Discussed offline, let's put "_" for every customObject